content/learning-paths/microcontrollers/img_nn_stcube/build_nn.md

```diff
@@ -156,26 +169,24 @@ model.add(Dropout(0.5))
 model.add(Dense(10)) #The number of classes
 model.add(Activation('softmax'))
 ```
-Execute the code blocks below to compile and train the model. This training uses only one epoch for time constraint, but will still take ~15 minutes to execute.
+
+Execute the code blocks below to compile and train the model. If tens of epochs are used, the training could take more than 10 hours because the dataset has 50,000 training images. Therefore, the model trained for 50 epochs is provided for testing (File: ‘Data/models/cifar10_model.h5’). You can use the model if you don't have enough time to train your own model. 
```

Comment:
We try to avoid conditions like "could". We try to use the present tense wherever that is possible. So here this might instead be "If tens of epochs are used, the training can/might take more than..." Choose which option you prefer.

Reply:
Thanks! Done